services/asset/src/lib.rs

```diff
@@ -579,10 +570,8 @@ impl<SDK: ServiceSDK> AssetService<SDK> {
     }
 
     #[cfg(test)]
```

Comment:
I blv we could remove this /test/ function due to that we have a [read] function get_admin.

pls remove this can fix the test if needed